crates/katana/core/src/service/messaging/starknet.rs

```diff
@@ -200,7 +204,7 @@ impl Messenger for StarknetMessaging {
             .await
             .map_err(|_| Error::SendError)
             .unwrap()
-            .iter()
+            .into_iter()
             .for_each(|(block_number, block_events)| {
                 debug!(
```

Comment:
If we switch to a simple `Vec`, then we will print a debug statement for each event being processed. 👍